examples/modules/TMCM6214/CANopen/rotate_demo_using_sdo_2-axis.py

```diff
@@ -0,0 +1,58 @@
+"""
+Uses SDO to rotate  motor1 and motor3 for 5 seconds in profile velocity mode.
+
+Tested with:
+* Firmware version 3.22
+* Using the Python canopen package version 2.0.0
+"""
+import time
+import canopen
+from pytrinamic.modules.canopen_node import TmcmNode
+
+with canopen.Network() as network:
+    network.connect(channel='PCAN_USBBUS1', bustype='pcan', bitrate=1_000_000)
+
+    tmcm_6214 = TmcmNode(1, 'TMCM-6214.eds')
+    motor=[1,3]
```

Comment:
nice idea to have a central motor mapping list 👍 , this way one does not need to change the code below if motors are attached to a different board output